src/net.cpp

```diff
@@ -715,9 +731,9 @@ bool CNode::ReceiveMsgBytes(const char* pch, unsigned int nBytes)
         if (handled < 0)
                 return false;
 
-        // BU: only reject the message if it is some multiple of the excessive 
+        // BU: only reject the message if it is some multiple of the excessive
         // block size.  Since traffic shaping will keep the bandwidth in check
-        // this basically eliminates nodes that are deliberately trying to screw us up. 
+        // this basically eliminates nodes that are deliberately trying to screw us up.
```

Comment:
While I indeed agree that trailing space are a sign of lack of attention to detail that ought to go in a codebase as sensitive as bitcoin's, it is worth it's own diff rather than being mixed with this one.